### ISSUE TITLE
test

### ISSUE THREAD

**Issue by joe-chiu** [mkjunior19/mkangista#1]
test

**Comment by mkjunior19** [comment]
got an email for this.

On Wed, Sep 16, 2015 at 1:49 PM, joe-chiu notifications@github.com wrote:

> test
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mkjunior19/mkangista/issues/1.

## 

Maria

**Comment by joe-chiu** [comment]
Interesting, if I respond to you, where can you see it? See the email address above.

-Joe

**Comment by joe-chiu** [comment]
You mean this address?

From: Joe Chiu
Sent: Wednesday, September 16, 2015 4:35 PM
To: mkjunior19/mkangista reply@reply.github.com
Cc: mkjunior19/mkangista mkangista@noreply.github.com; Maria Kang jkang@microsoft.com
Subject: RE: [mkangista] test (#1)

Interesting, if I respond to you, where can you see it? See the email address above.

-Joe

**Comment by joe-chiu** [comment]
Yes, do you get the reply in the email (or in any ways on the github site)?

From: Maria Kang
Sent: Wednesday, September 16, 2015 4:36 PM
To: Joe Chiu Yuan-Chun.Chiu@microsoft.com; mkjunior19/mkangista reply@reply.github.com
Cc: mkjunior19/mkangista mkangista@noreply.github.com
Subject: RE: [mkangista] test (#1)

You mean this address?

From: Joe Chiu
Sent: Wednesday, September 16, 2015 4:35 PM
To: mkjunior19/mkangista reply@reply.github.com
Cc: mkjunior19/mkangista <mkangista@noreply.github.com<mailto:mkangista@noreply.github.com>>; Maria Kang <jkang@microsoft.com<mailto:jkang@microsoft.com>>
Subject: RE: [mkangista] test (#1)

Interesting, if I respond to you, where can you see it? See the email address above.

-Joe

**Comment by joe-chiu** [comment]
[cid:image001.png@01D0F09E.6B8A9E50]

[cid:image002.png@01D0F09E.6B8A9E50]

From: Joe Chiu
Sent: Wednesday, September 16, 2015 4:38 PM
To: Maria Kang jkang@microsoft.com; mkjunior19/mkangista reply@reply.github.com
Cc: mkjunior19/mkangista mkangista@noreply.github.com
Subject: RE: [mkangista] test (#1)

Yes, do you get the reply in the email (or in any ways on the github site)?

From: Maria Kang
Sent: Wednesday, September 16, 2015 4:36 PM
To: Joe Chiu <Yuan-Chun.Chiu@microsoft.com<mailto:Yuan-Chun.Chiu@microsoft.com>>; mkjunior19/mkangista reply@reply.github.com
Cc: mkjunior19/mkangista <mkangista@noreply.github.com<mailto:mkangista@noreply.github.com>>
Subject: RE: [mkangista] test (#1)

You mean this address?

From: Joe Chiu
Sent: Wednesday, September 16, 2015 4:35 PM
To: mkjunior19/mkangista reply@reply.github.com
Cc: mkjunior19/mkangista <mkangista@noreply.github.com<mailto:mkangista@noreply.github.com>>; Maria Kang <jkang@microsoft.com<mailto:jkang@microsoft.com>>
Subject: RE: [mkangista] test (#1)

Interesting, if I respond to you, where can you see it? See the email address above.

-Joe
